TileStache/Goodies/VecTiles/transform.py

```diff
@@ -2701,3 +2701,22 @@ def add_iata_code_to_airports(shape, properties, fid, zoom):
         properties['iata'] = iata_code
 
     return shape, properties, fid
+
+
+def normalize_leisure_kind(shape, properties, fid, zoom):
```

Comment:
If we're normalizing them all to `fitness` should this be `normalize_fitness_kind` instead?